lib/matplotlib/cbook/__init__.py

```diff
@@ -35,7 +35,7 @@
     _rename_parameter, _delete_parameter, _make_keyword_only,
     _deprecate_method_override, _deprecate_privatize_attribute,
     _suppress_matplotlib_deprecation_warning,
-    MatplotlibDeprecationWarning, mplDeprecation)
+    MatplotlibDeprecationWarning, mplDeprecation, _classproperty)
```

Comment:
(To match other change, unless you want to fix other callers.)
```suggestion
    MatplotlibDeprecationWarning, mplDeprecation, classproperty as _classproperty)
```